opam

```diff
@@ -34,9 +34,11 @@ remove: [
 depends: [
   "ocamlfind" {build}
   "re"
-  "sexplib" {>="109.53.00"}
+  "sexplib" {>= "113.24.00"}
```

Comment:
I did not question the existance of the sexplib dependency.

For versions: I've a working setup with sexplib-113.00.00 and ppx_sexp_conv-113.09.00, which makes it coexist with other packages depending on sexplib <= 113.00.00 (due to sexplib.syntax/camlp4).  Only ppx_sexp_conv-113.24.00 introduced a dependency onto sexplib >= 113.24.00